packages/backend/src/api/controllers/UserController.ts

```diff
@@ -81,16 +84,29 @@ export class UserController {
       this.userTransactionRepository.getCountByStarkKey(starkKey),
     ])
 
+    let assetDetailsMap = {}
```

Comment:
Could we type it as `Record<string, AssetDetails>` here?

Reply:
Done